### ISSUE TITLE
Can I use ALRA imputed matrix to perform the `Find Marker Gene` step ?

### ISSUE THREAD

**Issue by goushixue** [KlugerLab/ALRA#10]
Hello, as the title say, can I use the imputed matrix to find marker genes and differentially expressed genes for downstream analysis ?

**Comment by JunZhao1990** [comment]
Yes, you can run "Find Marker Gene" step after ALRA in the downstream analysis.

**Comment by yamihn** [comment]
@JunZhao1990 when we set alra assay to find markers Is very hard to distinguish Cell populations
